container-guide/adoc/containers-podman-overview.adoc

```diff
@@ -4,33 +4,67 @@ include::attributes-product.adoc[]
-
-Moving from {deng} to {podman} does not require any changes in the established workflow. There is no need to rebuild images, and you can use the exact same commands to build and manage images as well as run and control containers.
-
-{podman} differs from {deng} in two important ways.
-
-* {podman} does not use a daemon, so the container engine interacts directly with an image registry, containers, and an image storage. As {podman} does not have a daemon, it provides integration with systemd. This makes it possible to control containers via systemd units. You can create these units for existing containers as well as generate units that can start containers if they do not exist in the system. Moreover, {podman} can run systemd inside containers.
-* Because {podman} relies on namespaces, which provide an isolation mechanism for Linux processes, it does not require root privileges to create and run containers. This means that {podman} can run in root mode as well as in an unprivileged environment. Moreover, a container created by an unprivileged user cannot get higher privileges on the host than the container's creator.
-
-[[cha-podman-install]]
+https://podman.io/[Podman] is short for Pod Manager Tool. It is a daemonless container engine for
+managing Open Container Initiative (OCI) containers on a Linux system, and it offers a drop-in
+alternative for {docker}.
+{podman} can be used to create OCI-compliant container images using a `Dockerfile` and a range of
+commands identical to {deng}. For example, the `podman build` command performs the same task as
+`docker build`. In other words, {podman} provides a drop-in replacement for {deng}.
+
+Moving from {deng} to {podman} does not require any changes in the established workflow. There is no
+need to rebuild images, and you can use the exact same commands to build and manage images as well
+as run and control containers.
+
+{podman} differs from {deng} in four main ways:
+
+* {podman} does not use a daemon, so the container engine interacts directly with an image registry,
+containers, and an image storage when needed.
+* {podman} has a native systemd integration. Allowing to control containers via systemd units which
+can be created for existing containers as well as generate units that can start containers if they
+do not exist in the system. Moreover, {podman} can run systemd inside containers.
+* Because {podman} relies on namespaces, which provide an isolation mechanism for Linux processes,
+it does not require root privileges to create and run containers. This means that {podman} can run
+in root mode as well as in an unprivileged environment. Moreover, a container created by an
+unprivileged user cannot get higher privileges on the host than the container's creator.
```

Comment:
I think this is a tad bit misleading. Every container runtime/engine to an extent would need to be able to create namespaces to spin up a container but the problem is that in order to create a namespace, you've to have root privileges. Docker was able to do this without you doing `sudo docker ...` because it's the docker daemon that does the heavy lifting and it always runs as root(so far). So if an attacker gets hold of the system even without root access, they can make the docker engine (which runs as root) do malicious stuff via the docker cli which is problematic.

Podman, otoh, allows you to run containers even if you're a non-privileged user (non-root) and that's a welcome feature in terms of security.

Reply:
Or did you mean user namespaces specifically and not the catch-all term namespaces?

Reply:
hummm you are right, I think I will just remove the first sentence completely.

Reply:
Fixed with https://github.com/SUSE/doc-unversioned/pull/53/commits/e96bf7a155186d7a279b705ee24fb5207c05f643